spec/bolt_spec/plan_spec.rb

```diff
@@ -315,6 +315,59 @@ def expect_action
     end
   end
 
+  context 'with out::verbose' do
+    let(:plan_name) { 'plans::out_verbose' }
```

Comment:
This is expecting to find a plan in `fixtures/bolt_spec/plans/plans/out_verbose.pp`.

Reply:
Great catch! Thank you :)